component/slos.libsonnet

```diff
@@ -46,6 +47,7 @@ local prometheusRule(name) =
         } for r in g.rules ],
       }
       for g in super.groups
+      if !(alertsDisabled && std.startsWith(g.name, 'sloth-slo-alerts'))
```

Comment:
such thing is much clearer for me and it doesn't introduce superfluous variables

Reply:
this will not even compile and you need to add `== false` that's why I came up with alertsDisabled. It's easier to read the already complicated if statement.

Reply:
I don't like this solution, but I trust You picked the best option